packages/twenty-server/src/engine/guards/demo.env.guard.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  Injectable,
+  ExecutionContext,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+
+import { EnvironmentService } from 'src/integrations/environment/environment.service';
+import { getRequest } from 'src/utils/extract-request';
+
+@Injectable()
+export class DemoEnvGuard extends AuthGuard(['jwt']) {
+  constructor(private readonly environmentService: EnvironmentService) {
+    super();
+  }
+
+  getRequest(context: ExecutionContext) {
+    return getRequest(context);
+  }
+
+  handleRequest(err: any, user: any) {
```

Comment:
can we type this?

Reply:
@charlesBochet The function comes from the interface `AuthGuard`. In there the types are `any`. I did not manage to override those types without having the linter complaining